java/spring/container/Application/azure-pipelines.yml

```diff
@@ -7,10 +7,14 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 variables:
-  imageName: 'javasample:$(build.buildId)'
+  imageVersion: '$(ImageName):$(build.buildId)'
```

Comment:
ImageVersion makes me think of the buildId alone instead of ImageName and ImageVersion. Maybe we can call it ImageNameAndTag.